### DISCUSSION TITLE
Tab separated CSV report

### DISCUSSION THREAD

**Discussion by TheRealStriff** [Barbarur/NovaPoint#21] [Ideas]
Would it be possible to do tab separated CSV reports.

Looks like a lot of users use commas in the file names.
Which makes a bit of a mess in the report to align to the headers.

Tab separated csv I would think should sort this?

Cheers

**Comment by Barbarur** [comment]
I'm checking alternatives for the CSV file which are as easy to use, but without this character-type issues and potentially lighter.

As temporal solution I will add quotation marks for each value so commas inside name won't make this type of problem. I should be able to add it on the next release.

**Reply by TheRealStriff** [reply]
Great, adding Double quotes would certainly help!
Thanks again

**Comment by Barbarur** [comment]
This should be fixed from release 0.7.2

**Comment by TheRealStriff** [comment]
Confirming using double quotes for the logs works great.

Thanks for this
